test/builtin-ppx-private.t

```diff
@@ -0,0 +1,29 @@
+Test to showcase errors when using %%private extensions with melange ppx
+
+  $ . ./setup.sh
+  $ cat > dune-project <<EOF
+  > (lang dune 3.7)
+  > (using melange 0.1)
+  > EOF
+
+  $ cat > dune <<EOF
+  > (melange.emit
+  >  (target output)
+  >  (alias mel)
+  >  (emit_stdlib false)
+  >  (preprocess (pps melange.ppx)))
+  > EOF
+
+  $ cat > main.ml <<EOF
+  > (**
+  >   Hey
+  > *)
+  > [%%private
+  >   external fromMouseEvent :
+  >     Webapi.Dom.MouseEvent.t -> Webapi.Dom.Event.t = "%identity"]
+  > EOF
+
+  $ dune build @mel
+  File "main.ml", lines 1-3:
+  Error: Attributes not allowed here
```

Comment:
this looks like a ppxlib limitation. https://github.com/ocaml-ppx/ppxlib/blob/main/src/extension.ml#L122-L127

it works if you add the attributes in the external rather than the extension.